### DISCUSSION TITLE
HTML/XSS Sanitizer?

### DISCUSSION THREAD

**Discussion by tmikaeld** [denoland/deno#17206] [Q&A]
Are there any HTML sanitizer for Deno? 
I tried getting dompurify and sanitize-html to work in Deno, but it won't import them, even when using esm.sh.

Update: I see https://github.com/lucacasonato/ammonia-wasm is made by @lucacasonato, are there plans to update it or add it native to Deno?

**Comment by ggstuart** [comment]
Late reply but does the [escape function](https://jsr.io/@std/html/doc/~/escape) from the standard library work for you?
